recipes/bleach/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "bleach" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "978e758599b54cd3caa2e160d74102879b230ea8dc93871d0783721eef58bc65" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - html5lib >=0.999,!=0.9999,!=0.99999,<0.99999999
+    - six
+
+test:
+  imports:
+    - bleach
+
+about:
+  home: https://github.com/mozilla/bleach
+  license: Apache
```

Comment:
What version of Apache?

Also could optionally add `license_family: Apache` IIRC.